src/main/java/org/pac4j/jax/rs/pac4j/JaxRsApplicationLogoutLogic.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.jax.rs.pac4j;
+
+import org.pac4j.core.engine.DefaultApplicationLogoutLogic;
+
+/**
+ * @author Victor Noel - Linagora
+ * @since 1.1.1
+ * 
+ */
+public class JaxRsApplicationLogoutLogic<C extends JaxRsContext> extends DefaultApplicationLogoutLogic<Object, C> {
```

Comment:
I don't know… people already have access to the logic via a getter, we don't really need the filter to be aware of this (in terms of public API), no?
I think all of this should be the responsibility of the logic: the filter should only be able to:
 - set a default implementation for the profile manager factory, if needed, as part of its internal implementation
 - have a getter and a setter for the logic

That should be enough